prow/config/external_plugins_config.yaml

```
@@ -1613,7 +1613,7 @@ ti-community-format-checker:
     required_match_rules:
       - pull_request: true
         title: true
-        regexp: ""
+        regexp: "^.{1,40}: .{1,160}$"
```

建议增加拦截以 `.` 结尾的 title
```suggestion
        regexp: "^.{1,40}: .{1,160}[^\.]$"
```

---

测试好后，可以后面单独一个 PR 修改哈。